frontend/src/concepts/connectionTypes/utils.ts

```diff
@@ -372,7 +372,8 @@ export const filterModelServingConnectionTypes = (
     })
     .filter((t) => t != null);
 
-export const VALID_ENV_VARNAME_REGEX = /^[A-Za-z_][A-Za-z0-9_-]*$/;
+// eslint-disable-next-line no-useless-escape
+export const VALID_ENV_VARNAME_REGEX = /^[A-Za-z_][A-Za-z0-9_\-\.]*$/;
```

Comment:
You don't need to escape `.` chars inside a character class (i.e. inside square brackets). This is what the linter is trying to warn you about.
```suggestion
export const VALID_ENV_VARNAME_REGEX = /^[A-Za-z_][A-Za-z0-9_\-.]*$/;
```

Reply:
@alexcreasy Done